solr/solrj/src/java/org/apache/solr/client/solrj/impl/Http2SolrClient.java

```diff
@@ -128,7 +128,7 @@ public class Http2SolrClient extends SolrClient {
   private static final List<String> errPath = Arrays.asList("metadata", "error-class");
 
   private HttpClient httpClient;
-  private volatile Set<String> queryParams = Collections.emptySet();
+  protected Set<String> queryParams = Set.of();
```

Comment:
I think this should remain 'volatile', see @dsmiley's comments on the builder's use of volatile.

Reply:
Thanks for the sharp eyes.. yeah, I got moving too fast.